src/Tizen.NUI/src/public/XamlBinding/Setter.cs

```diff
@@ -40,10 +40,23 @@ public sealed class Setter : IValueProvider
         [EditorBrowsable(EditorBrowsableState.Never)]
         public BindableProperty Property { get; set; }
 
+        private bool isOriginalValue = false;
+        private object value;
         /// <since_tizen> 6 </since_tizen>
         /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public object Value { get; set; }
+        public object Value
```

Comment:
please remove this inappropriate comment =>

        /// <since_tizen> 6 </since_tizen>
        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.